uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup/leave-request-popup-comments-tab.html

```diff
@@ -1,42 +1,44 @@
-<div
-  class="text-center chr_leave-request-modal__comments_empty"
-  ng-show="commentsCtrl.getActiveComments().length === 0">
-  No comments available. Comments can be added below.
-</div>
-<div
-  class="chr_leave-request-modal__comments"
-  ng-hide="commentsCtrl.getActiveComments().length === 0">
+<crm-loading show="!$ctrl.loading.loggedInContactId">
```

Comment:
I am not quite sure in this. Can we do smth like vm.loading.tab and assign it to false once vm.loading.loggedInContactId is set, but in the controller? Otherwise, I read it like "load until user logs in" (or is just me?)